jest.config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "collectCoverage": false,
+  "coverageReporters": ["lcov", "text"],
+  "collectCoverageFrom": ["**/src/**/*.ts", "**/config/**/*.ts"],
+  "preset": "ts-jest",
+  "testMatch": ["**/*.spec.ts"],
```

Comment:
It will be necessary to change the testMatch here because of this commit https://github.com/khaosdoctor/gotql/pull/46/commits/a2b01df3ee2a3a22bfb50dd7cd0b663da44e6bfb

from: *.spec to *.test

Suggestion:
`"testMatch": ["**/tests/**/*.test.ts"]`

Reply:
Good Point

```suggestion
  "testMatch": ["**/*.test.ts"],
```